prelude/slang-cpp-prelude.h

```diff
@@ -272,6 +274,13 @@ struct ISlangUnknown
     virtual SLANG_NO_THROW uint32_t SLANG_MCALL release() = 0;
 };
 
+#define SLANG_COM_INTERFACE(a, b, c, d0, d1, d2, d3, d4, d5, d6, d7) \
```

Comment:
Do we need to guard, with #ifndef? Might be problem if prelude is included as well as "slang.h", or give it a prelude exclusive name like SLANG_PRELUDE_COM_INTERFACE